src/care/LoopFuser.h

```diff
@@ -916,14 +922,14 @@ void LoopFuser<REGISTER_COUNT, XARGS...>::registerAction(const char * fileName,
             if (verbose) {
                printf("hit reserved flushActions\n");
             }
-            flushActions();
+            flush_now = true;
          }
          // if we are approaching the 2^31-1 limit proactively flush
```

Comment:
This comment is no longer appropriate.

Reply:
@liu15, looks like you changed this comment on a different branch?

Reply:
Yeah, the other branch is a bugfix for this.